### PR TITLE
Add  `--config-file` argument to process custom configuration files

### PR DESCRIPTION
**Pull request by pmbaldha**
Fixes https://github.com/wp-cli/ideas/issues/138

### CODE REVIEW

**Comment by schlessera** [comment]
@pmbaldha Can you add Behat tests to this PR?

**Comment by pmhost9** [comment]
@schlessera I am developing behat tests for it.

**Comment by pmhost9** [comment]
@schlessera I have written all related behat tests.

**Comment by pmbaldha** [comment]
@schlessera I have written all related behat tests.

File: src/Config_Command.php [hunk 1]
```diff
@@ -182,14 +202,24 @@ public function create( $_, $assoc_args ) {
...
+            if ( ! file_exists( $path ) ) {
+                WP_CLI::error( "'{$wp_config_file_name}' not found.\nEither create one manually or use `wp config create`." );
+            }
+        } else {
+            $path                = $this->get_config_path();
+            $wp_config_file_name = 'wp-config.php';
+		}
+
+		$contents            = file_get_contents( $path );
+		$r                   = Utils\launch_editor_for_input( $contents, $wp_config_file_name, 'php' );
```
[the remark this answers is not in the document]
Comment (pmbaldha) [reply src/Config_Command.php:196]: @schlessera These spaces are part of WP bonding standard. I have tried to remove these spaces but PHPCBF command have put it again.

**Review by schlessera — APPROVED** [review]

**Pull request merged**